tracer/src/Datadog.Trace/Ci/Agent/MessagePack/CIEventMessagePackFormatter.cs

```diff
@@ -123,40 +77,39 @@ public override int Serialize(ref byte[] bytes, int offset, EventsPayload? value
         private ArraySegment<byte> GetEnvelopeArraySegment()
         {
             var offset = 0;
-            var bytes = new byte[1024];
+            var bytes = new byte[512];
 
             offset += MessagePackBinary.WriteMapHeader(ref bytes, offset, 3);
 
-            // .
+            // # Version
 
+            // Key
             offset += MessagePackBinary.WriteStringBytes(ref bytes, offset, VersionBytes);
+            // Value
```

Comment:
The schema changed but the version was not bumped?

Reply:
right!, because this feature has not reached neither GA or private beta. So we decided to do it as it.